runbooks/data/deprecation-stored-files.md

```diff
@@ -0,0 +1,19 @@
+# Deleting / deprecating files in Google Cloud Storage
+
+Occasionally, we want to assess our Google Cloud Storage buckets for outdatedness and deprecate buckets that we have no use for. Please follow the steps outlined here when performing a deletion or deprecation.
+
+1. In the Google Cloud Console, sort [our available GCS buckets](https://console.cloud.google.com/storage/browser) by their Last Modified date, in ascending order.
+
+2. Among buckets not recently modified (more than ~3 months since the last update), there are two general categories:
+    * Buckets used for testing, prefixed with "test-", generally correspond to infrequent tests of Airflow jobs and other scripts that take place during new feature development. The buckets themselves should generally remain in existence (unless the corresponding job/script is no longer actively used in production), but any objects they contain from previous rounds of testing can be deleted.
```

Comment:
They should all be pretty easy to re-populate. RT will be re-populated continuously via the test archiver but the rest should be easily populated via local Airflow execution.

Reply:
Since there's no universal here, I added a note about this in [49b5007](https://github.com/cal-itp/data-infra/pull/2845/commits/49b5007f31dde478f75ae59d7724fec4308afb80) essentially saying "hey, think about whether you'll need to repopulate raw artifacts later if you delete them for a given test bucket".